test/extended/util/image/image.go

```diff
@@ -239,3 +251,47 @@ func GetMappedImages(originalImages map[string]k8simage.ImageID, repo string) ma
 	}
 	return configs
 }
+
+func GetPullSpecFor(image string) (string, bool) {
+	if spec, found := releasePullSpecRemoteMirror[image]; found {
```

Comment:
this will not be populated if ` InitializeReleasePullSpecString` was not called, and currently that init only gets called when there's no optional capabilities(hence no registry) on the cluster.  

For the way this is currently used (it's only called for images that have a valid pullthrough pullspec) that may be ok, but is there a reason not to just always call the init and future proof later usage of this on clusters that have a registry, and for which the image string is not one of `tools` or `cli`

Reply:
Yeah I agree, I was trying to be conservative with changing up how things get populated during normal runs, I went ahead and updated to always be called and only make a decision to replace the existing `cli` `tools` pullspec if ImageRegistry is not present. Thanks for the input!